index.js

```diff
@@ -1,34 +1,48 @@
 'use strict';
 
+const defer = require('p-defer');
+
+class AbortError extends Error {
+	constructor(message) { // eslint-disable-line no-useless-constructor
+		super(message);
```

Comment:
I believe you can remove the `// eslint-disable-line no-useless-constructor` as well now.